schema/ContainerImage.schema.yaml

```diff
@@ -0,0 +1,84 @@
+title: ContainerImage
+'@id': stencila:ContainerImage
+extends: SoftwareApplication
+role: primary
+status: experimental
+category: code
+description: A linux container image (such as provided by Docker or Singularity) that is used in scientific computing.
+properties:
+  annotations:
+    '@id': stencila:annotations
+    description: | 
+      One or more annotations (labels) for the container image, typically key value pairs represented as strings
+    aliases:
+      - labels
+    type: array
+    items:
+      anyOf:
+        - type: string
+        - $ref: PropertyValue
+  applicationCategory:
+    '@id': stencila:applicationCategory
+    description: A container image is the binary that empowers Linux containerization
+    enum: ['Linux containerization']
+  containerRuntimePlatform:
+    '@id': stencila:containerRuntimePlatform
+    description: The container technology (e.g., Docker or Singularity)
+    type: string
+  config:
+    '@id': stencila:config
+    description: The container configuration, typically dumped json in the case of a manifest or other text
+    aliases:
+      - manifest
+    type: string
+  configMediaType:
+    '@id': stencila:configMediaType
+    description: The media type for an associated ContainerImage config
```

Comment:
Are these values same as the `imageMediaType`s? As someone unfamiliar with the spec, an example or link to valid options would be greatly appreciated.

Reply:
Yes and no - they are different (e.g. image manifest vs layer) but all come from the same list in the image spec: https://github.com/opencontainers/image-spec/blob/master/media-types.md.